docs/BasicCommunication/OnSystemCapabilityUpdated/index.md

```diff
@@ -0,0 +1,82 @@
+## OnSystemCapabilityUpdated
+
+Type
+: Notification
+
+Sender
+: SDL
+
+Purpose
+: Inform the HMI that a specific system capability has changed
```

Comment:
This is being formatted as a table, but incorrectly. Please review https://smartdevicelink.com/en/docs/pull_request/a4c08d3a7aa73f06394196098b108f3c9f8bac09/basiccommunication/onsystemcapabilityupdated/ and revise.

Reply:
Please refer to comment https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/139#discussion_r276242113